### DISCUSSION TITLE
Is there a way to call one of pyscript's internal functions from python?

### DISCUSSION THREAD

**Discussion by SquarishRectangle** [pyscript/pyscript#1354] [Q&A]
Context: #1351 
After spending some time digging through pyscript's source code, I think I've found where Event handlers are initialized ([initHandlers](https://github.com/pyscript/pyscript/blob/b61e8435d13e2f4836781805794bc007a76511aa/pyscriptjs/src/components/pyscript.ts#L169)). Is there a way to call this function from pyscript itself?

Example of what I would ideally want to do:
```
from pyscript import initHandlers, interpreter
initHandlers(interpreter)
```

**Comment by hoodmane** [comment]
I think you can do:

```py
from js.pyscript import whatever
```

**Reply by SquarishRectangle** [reply]
No luck with that :(. initHandlers isn't in js.pyscript either.
